docs/security-and-privacy-design/netdata-agent-security.md

```diff
@@ -10,6 +10,11 @@ databases, sent to upstream Netdata servers, or archived to external time-series
 
 ## User Data Protection
 
+> **Note**
+>
+> With Netdata's decentralised / distributed architecture the data and metrics are always stored locally (on the user's host), it is the user's responsibility to backup, recover and ensure his data is available.
```

Comment:
```suggestion
> **Note**
>
> Users are responsible for backing up, recovering, and ensuring their data's availability because Netdata stores data locally on each system due to its decentralized architecture.
```